File: yongkeun/4week/영어가싫어요.js
function solution(numbers) {
  let number = [
    "zero",
    "one",
    "two",
    "three",
    "four",
    "five",
    "six",
    "seven",
    "eight",
    "nine",
  ]; //배열의 index를 result 값에 추가
  // .replace(SearchValue, NewValue) : SearchValue를 찾아 NewValue로 바꿔줌
  for (let i = 0; i < 8; i++) {
    //for문으로 한번 더 감싸주지 않으면 "twozerozero" 같은 zero가 여러번 나오는 경우에는 가장 처음 zero만 0으로 바뀌므로 for문 한번 더 씀
    number.forEach((value, index) => {
      numbers = numbers.replace(value, index);
    });
  }
  return Number(numbers);
}

File: dongmin/week4/영어가 싫어요.java
class Solution {
    public long solution(String numbers) {
        long answer = 0;
        String[] strNumber = {"zero",
                "one", "two", "three",
                "four", "five", "six",
                "seven", "eight", "nine"
        };      //각각에 문자열 숫자들을 찾을 배열 strNumber를 선언
        for (int i = 0; i < strNumber.length; i++) {
            if (numbers.contains(strNumber[i]))     //문자열에서 strNumber 배열에 해당하는 문자열을 찾으면 true
                numbers=numbers.replaceAll(strNumber[i],(i)+"");    //numbers문자열에 strNumber[i]에 해당하는 값을 
                                                                    //현재 i값으로 바꿈(i는 int형이기 때문에 공백 문자를 더해 문자열로 만들어줌)
        }


        answer = Long.parseLong(numbers);       //numbers를 long형으로 바꿔서 answer에 넣어줌
        return answer;
    }
}